webapp/src/routes/[[lang]]/my/organizations/[id]/verification-flows/_partials/verificationFlowForm.svelte

```diff
@@ -223,7 +223,7 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 			title={m.Advanced_settings()}
 			description={m.advanced_settings_description()}
 		/>
-		<Checkbox field="public" {superform}>
+		<Checkbox field="public" {superform} options={{ disabled: true }}>
```

Comment:
@andrea-dintino can you double check that this is ok? I have a déjà vu about this ;p If the review is okay, approve it. I will take care of the merge and deploy!